enterprise/tpa_pipeline.py

```diff
@@ -78,7 +98,7 @@ def active_provider_enforces_data_sharing(request, enforcement_location):
         enforcement_location (str): the point where to see data sharing consent state.
         argument can either be "optional", 'at_login' or 'at_enrollment'
     """
-    running_pipeline = request.session.get('partial_pipeline')
+    running_pipeline = request.session.get('partial_pipeline_token')
```

Comment:
@haikuginger I would say it's sufficient: we use the token to get the pipeline anyway, and getting the pipeline is thus wholly dependent on that. If the token doesn't exist, we wouldn't have gotten the pipeline anyway.